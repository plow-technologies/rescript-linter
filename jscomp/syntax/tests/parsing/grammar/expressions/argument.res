let foo = (~a) => a(. ()) +. 1.
let a = (. ()) => 2
let bar = foo(~a)
let comparisonResult = compare(. currentNode.value, ~targetValue)
callback(. firstNode, ~y)

document.createElementWithOptions(. "div", elementProps(~onClick=_ =>
    Js.log("hello world")
  ))


resolve(.)
resolve(. ())
