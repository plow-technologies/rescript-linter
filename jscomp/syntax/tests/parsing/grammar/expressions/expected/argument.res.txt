let foo ~a:((a)[@res.namedArgLoc ])  =
  ((a (let __res_unit = () in __res_unit))[@res.uapp ]) +. 1.
let a = ((Function$ (fun () -> 2))[@res.arity 1])
let bar = foo ~a:((a)[@res.namedArgLoc ])
let comparisonResult =
  ((compare currentNode.value ~targetValue:((targetValue)[@res.namedArgLoc ]))
  [@res.uapp ])
;;((callback firstNode ~y:((y)[@res.namedArgLoc ]))[@res.uapp ])
;;((document.createElementWithOptions {js|div|js}
      (elementProps ~onClick:((fun _ -> Js.log {js|hello world|js})
         [@res.namedArgLoc ])))[@res.uapp ])
;;((resolve ())[@res.uapp ])
;;((resolve (let __res_unit = () in __res_unit))[@res.uapp ])